app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -362,7 +380,7 @@ class SamplesHeatmapView extends React.Component {
     // If there are failed/waiting samples selected, display a warning
     // to the user that they won't appear in the heatmap.
     if (sampleValidationInfo.invalidSampleNames.length > 0) {
-      this.showNotification();
+      this.showNotification("invalid samples");
```

Comment:
Define a constant for these strings... it easy to make typos on these.

Reply:
Done!